index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awesome Books</title>
+    <link rel="stylesheet" href="./css-file/style.css" />
+  </head>
+  <body>
+    <header id="navlink-header"></header>
+    <div id="dateTime"></div>
+    <div id="top-part">
+      <h1>All Awesome Books</h1>
+
+      <div id="displayField"></div>
+    </div>
+    <form id="form" class="form">
+      <div id="container">
+        <div id="inputdiv">
+          <h2>Add new Book</h2>
+          <input
+            type="text"
+            placeholder="Title"
+            class="titleName"
+            id="titleName"
+          />
+          <input
+            type="text"
+            placeholder="Author"
+            class="authorName"
+            id="authorName"
+          />
+          <button class="btn" type="submit">Add</button>
+        </div>
+      </div>
+    </form>
+    <div id="contact">
+      <h2>Contact Information</h2>
+      <span> contact us for anything, we are here to help. </span>
+      <ul>
+        <li>Our Email: Madetunji1376@gmail.com & ambrisexyz@gmail.com</li>
+        <li>Contact: +2349045637828</li>
+        <li>Address: 18 sulivan street, oxford road, oxford.</li>
+      </ul>
+    </div>
+    <footer id="footer">
+      <p>Copyright 2022 &copy;</p>
+    </footer>
+    <script type="module" src="webapp.js"></script>
```

Comment:
- [ ] As written by the project requirements. Link your `index.js` and `index.css` files to the `index.html`